star-rx72n-firmware/lib/rx_usb_comm/src/rx_usb_comm.c

```
@@ -754,6 +754,147 @@ rx_usb_comm_send_nack(rx_usb_comm_handle_t* handle, const uint16_t sequence, con
+  uint32_t wire_len = 0;
+
+  err = rx_frame_encode(&handle->encoder, &reset_ack_frame, handle->tx_buffer, &wire_len);
+  if (err != k_rx_ok) {
+    return err;
+  }
+
+  return rx_usb_write(handle->tx_buffer, wire_len);
+}
+
+rx_err_t rx_usb_comm_set_control_callbacks(rx_usb_comm_handle_t*    handle,
+                                            rx_usb_comm_control_cb_t on_ping,
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'rx_usb_comm_set_control_callbacks' of similar type ('int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)